cron/update_geolite_db.py

```diff
@@ -49,6 +49,8 @@ def save_db_data(save_path, db_data):
 def main():
     logger.debug('Starting')
     logger.debug(options)
+    if options.only_update_if_absent and os.path.isfile(options.db_location):
```

Comment:
I'm not sure we want to actually do this, I think we want to periodically update this file daily even if the file exists, because the ip -> country mapping can change and get out of date.

Reply:
This conditional triggers when *both* `options.only_update_if_absent` is `True` *and* the file exists. `options.only_update_if_absent` is a new command line argument which is used in the `@reboot` line of the `cron-entries.txt`.

As a result, this doesn't relate to the normal operation of the cron job, only the first run when the system comes up.